compositor_render/src/transformations/shader/validation.rs

```diff
@@ -0,0 +1,418 @@
+use naga::{ArraySize, Constant, ConstantInner, Handle, Module, ShaderStage, Type};
+
+#[derive(Debug, thiserror::Error)]
+pub enum ShaderValidationError {
```

Comment:
Error messages are inconsistent in the project in general. Sometimes we write full sentences starting with capital letters and ending with `.` and sometimes all lowercase.

Given that error messages sometimes will have more than one sentence the only consistent way I see to display them is to use full sentences starting with capital letters and `.` at the end.